package.json

```diff
@@ -7,9 +7,9 @@
     "build": "NODE_ENV=production lerna run build",
     "lint": "npm run lint:styles && npm run lint:scripts",
     "lint:fix": "npm run lint:styles:fix && npm run lint:scripts:fix",
-    "lint:styles": "stylelint packages",
+    "lint:styles": "stylelint --ignore-path .gitignore packages",
     "lint:styles:fix": "npm run lint:styles -- --fix",
-    "lint:scripts": "eslint --ext=js,jsx,ts,tsx .",
+    "lint:scripts": "eslint --ignore-path .gitignore --ext=js,jsx,ts,tsx .",
```

Comment:
I could've just added `storybook-static` to the .eslintignore and .stylelintignore files. It's kind of nice to just use the gitignore, though.

Reply:
yeah, that's awesome, full approve!